### ISSUE TITLE
Layout/EmptyLineAfterMagicComment should accept top-level `# shareable_constant_values` (Ruby 3.0)

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#9327]
The following should not raise an offense:

```ruby
# frozen_string_literal: true
# shareable_constant_values: literal

# Other comments
```

First two lines could be in the reverse order and that's ok too.
